openlibrary/templates/covers/book_cover.html

```diff
@@ -26,14 +26,14 @@
             </div>
     </div>
 $else:
-    <img src="$(cover_url or '/images/icons/avatar_book-sm.png')" height="58" title="$title" alt="We need a book cover for: $title"/>
+    <img src="$(cover_url or '/images/icons/avatar_book-sm.png')" srcset="cover_lg 2x" height="58" title="$title" alt="We need a book cover for: $title"/>
 
 <div class="hidden">
     <div class="coverFloat" id="seeImage">
         <div class="coverFloatHead">
             <h2>$:macros.TruncateString(title, 70)</h2>
             <a class="dialog--close">&times;<span class="shift">$_("Close")</span></a>
         </div>
-        <a class="dialog--close"><img src="$cover_lg" class="cover" alt="$title $_('by') $author_names"/></a>
+        <a class="dialog--close"><img src="$cover_lg" srcset="cover_lg 2x" class="cover" alt="$title $_('by') $author_names"/></a>
```

Comment:
This is already set to the larger image, so doesn't need a 2x version :+1:

```suggestion
        <a class="dialog--close"><img src="$cover_lg" class="cover" alt="$title $_('by') $author_names"/></a>
```